docs/MANUAL.md

```diff
@@ -150,6 +150,88 @@ kodelet conversation delete <conversation-id>
 kodelet conversation delete --no-confirm <conversation-id>
 ```
 
+## Shell Completion
+
+Kodelet provides shell completion support for bash, zsh, fish, and PowerShell. This enables tab completion for commands and flags, making the CLI experience more efficient.
+
+### Setup Instructions
+
+**Bash:**
+
+To load completions in your current shell session:
```

Comment:
Preferably loaded to the current user's shell config instead of /etc/completions dir.

E.g. ~/.zshrc ~/.bashrc